### ISSUE TITLE
touchScripts (windows7) does not work on unity3d5.12 32bit...

### ISSUE THREAD

**Issue by yty** [TouchScript/TouchScript#187]
win7Examples are running any touchscript.

log.
```
nitialize engine version: 5.1.2f1 (afd2369b692a)
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.0]
    Renderer: NVIDIA GeForce GTX 760 (ID=0x1187)
    Vendor:   NVIDIA
    VRAM:     1985 MB
Begin MonoManager ReloadAssembly
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\UnityEngine.dll (this message is harmless)
Loading E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\UnityEngine.dll into Unity Child Domain
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\Assembly-CSharp.dll (this message is harmless)
Loading E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\Assembly-CSharp.dll into Unity Child Domain
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\Assembly-UnityScript.dll (this message is harmless)
Loading E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\Assembly-UnityScript.dll into Unity Child Domain
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\UnityEngine.UI.dll (this message is harmless)
Loading E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\UnityEngine.UI.dll into Unity Child Domain
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\UnityEngine.Networking.dll (this message is harmless)
Loading E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\UnityEngine.Networking.dll into Unity Child Domain
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\UnityEngine.Analytics.dll (this message is harmless)
Loading E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\UnityEngine.Analytics.dll into Unity Child Domain
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\TouchScript.dll (this message is harmless)
Loading E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\TouchScript.dll into Unity Child Domain
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\TouchScript.Windows.dll (this message is harmless)
Loading E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\TouchScript.Windows.dll into Unity Child Domain
- Completed reload, in  0.045 seconds
desktop: 1920x1080 60Hz; virtual: 1920x1080 at 0,0
<RI> Initializing input.

<RI> Input initialized.

<RI> Initialized touch support.

Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\System.Core.dll (this message is harmless)
Platform assembly: E:\WorkFiles\Demo\test5.12Win7\test_Data\Managed\System.dll (this message is harmless)
WARNING: Shader Unsupported: 'Legacy Shaders/Diffuse' - All passes removed
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/coredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/.\coredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/coredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/coredll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/.\coredll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/coredll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/libcoredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/.\libcoredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/libcoredll.dll
No camera layers, adding CameraLayer for the main camera. (this message is harmless)
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/UnityEngineDebugBindings.gen.cpp Line: 65)

Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/coredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/.\coredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/coredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/coredll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/.\coredll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/coredll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/libcoredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/.\libcoredll.dll
Fallback handler could not load library E:/WorkFiles/Demo/test5.12Win7/test_Data/Mono/libcoredll.dll

```

**Comment by valyard** [comment]
Right, there's a dependency on coredll.dll for some reason.
What OS are you running this on?

**Comment by valyard** [comment]
Should be fixed in 6.8.

**Comment by yty** [comment]
@valyard
I use a win8.1 64bit  System ...

In addition, if you use touchScripts (windows8), no problem.
